db/dataset.py

```diff
@@ -205,7 +205,6 @@ def get_by_user_id(user_id, public_only=True):
             datasets.append(dict(row))
         return datasets
 
-
```

Comment:
you've deleted a line which already existed the PEP8 formatting guide suggests 2 newlines between functions. https://www.python.org/dev/peps/pep-0008/